syntax = "proto3";

service BooksService {
  rpc GetBookDetails(GetBookDetailsRequest) returns (BookDetailsResponse);
}

message GetBookDetailsRequest {
  string book_id = 1;
}

message BookDetailsResponse {
  string book_id = 1;
  string title = 2;
  string author = 3;
  string category = 4;
  string publisher = 5;
}